.github/workflows/scripts/detect_ts_ignore.py

```
@@ -0,0 +1,47 @@
+                        print(f"❌ Error: '@ts-ignore' found in {file} at line {line_num}")
+                        ts_ignore_found = True
+        except FileNotFoundError:
+            print(f"⚠️ Warning: File not found: {file}")
+        except OSError as e:
+            print(f"⚠️ Warning: Could not read {file}: {e}")
+        if not ts_ignore_found:
+            print("✅ No '@ts-ignore' comments found in the files.")
+
+    return 1 if ts_ignore_found else 0
+
+def main():
```

Ruff (0.8.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)